src/rt/sched/cown.h

```diff
@@ -50,9 +50,31 @@ namespace verona::rt
       YesTryFast
     };
 
-    Cown(const Descriptor* desc) : Object(desc)
+    Cown(bool initialise = true)
```

Comment:
Can this be a template parameter and then line 57 be a `constexpr` if?

Reply:
I was thinking about that, and then forgot.